src/content/peerconnection/munge-sdp/js/main.js

```diff
@@ -183,10 +185,8 @@ function onCreateSessionDescriptionError(error) {
 function setOffer() {
   var sdp = offerSdpTextarea.value;
   sdp = maybeAddLineBreakToEnd(sdp);
-  var offer = new RTCSessionDescription({
```

Comment:
we keep the original session description (including any extra things it may contain) around.